drivers/flash/flash_npcx_fiu_nor.c

```diff
@@ -273,31 +274,34 @@ static int flash_npcx_nor_erase(const struct device *dev, off_t addr,
 	}
 
 	/* size must be a multiple of sectors */
-	if ((size % config->min_erase_size) != 0) {
+	if ((size % BLOCK_4K_SIZE) != 0) {
 		LOG_ERR("Size %d is not a multiple of sectors", size);
 		return -EINVAL;
 	}
 
+	offset_64k_aligned = (addr % BLOCK_64K_SIZE) == 0;
+	offset_4k_aligned = (addr % BLOCK_4K_SIZE) == 0;
```

Comment:
Isn't this the same as `SPI_NOR_IS_SECTOR_ALIGNED`? Maybe we could have some variation of that one for 32k and 64k block?

Reply:
I updated the alignment condition as suggested in new revision. Thank you